ride_to_work_by_bike_config.toml

```diff
@@ -72,3 +72,4 @@ apiBase = "https://test.dopracenakole.cz/rest/"
 apiVersion = "1.0"
 
 urlApiLogin = "auth/login/"
+urlApiRefresh = "auth/refresh/tokens/"
```

Comment:
correct refresh token URL is auth/token/refresh/

Reply:
Because backend REST API Python lib was upgraded.

Reply:
Updated endpoint.